src/renderer/widgets/AddDelegationModal/components/DelegationCard.tsx

```diff
@@ -0,0 +1,99 @@
+import { useUnit } from 'effector-react';
+
+import { useI18n } from '@/app/providers';
+import { type DelegateAccount } from '@/shared/api/governance';
+import { cnTw } from '@/shared/lib/utils';
+import { BodyText, CaptionText, Card, FootnoteText, HeadlineText, Icon, Identicon, Truncate } from '@/shared/ui';
+import { AssetBalance } from '@/entities/asset';
+import { networkSelectorModel } from '@/features/governance';
+import { addDelegationUtils } from '../common/utils';
+
+type Props = {
+  delegate: DelegateAccount;
+};
+
+export const DelegationCard = ({ delegate }: Props) => {
+  const { t } = useI18n();
+
+  const chain = useUnit(networkSelectorModel.$governanceChain);
+
+  return (
+    <Card as="li">
+      <div className="flex flex-col gap-4">
+        <div className="flex gap-3">
+          {delegate.name ? (
+            <>
+              {delegate.image && !addDelegationUtils.isDefaultImage(delegate.image) ? (
+                <img src={delegate.image} alt={delegate.name} className={cnTw('w-11.5 h-11.5 rounded-full')} />
+              ) : (
+                <div
+                  className={cnTw(
+                    'w-11.5 h-11.5 flex items-center justify-center rounded-full',
+                    delegate.isOrganization ? 'bg-badge-orange-background-default' : 'bg-badge-background',
+                  )}
+                >
+                  {delegate.isOrganization ? (
+                    <Icon className="text-icon-warning" name="organization" />
+                  ) : (
+                    <Icon className="text-icon-accent" name="individual" />
+                  )}
+                </div>
+              )}
+            </>
+          ) : (
+            <Identicon address={delegate.accountId} size={46} />
+          )}
```

Comment:
A lot of ternary operators - a bit hard to read, maybe spread some into variables?